bundles/AdminBundle/Controller/Admin/MiscController.php

```diff
@@ -39,7 +40,10 @@
  */
 class MiscController extends AdminController
 {
-    /**
+    public function __construct (private AdminJsHelperService $adminJsHelperService)
```

Comment:
I would inject the service in the action (where it is needed) instead of the constructor.